### PR TITLE
DEV2-1677 check for tab-in in languages with tab indentation vs. spaces

### PR DESCRIPTION
**Pull request by dimacodota**
**problem:**
in the case of "tab" indentation (`\t`), the change detection was expecting only spaces `" "` and not tab indentation

### CODE REVIEW

File: src/binary/requests/tabSize.ts [hunk 1]
```diff
@@ -7,3 +7,6 @@ export default function getTabSize(): number {
   }
   return tabSize;
 }
+export function getTabsCount(): number {
+  return getTabSize() / getTabSize();
```
Comment (yairco1990) [inline src/binary/requests/tabSize.ts:11]: what am I missing? isn't that just 1?

File: src/test/suite/completion.test.ts [hunk 2]
```diff
@@ -178,19 +178,10 @@ describe("Should do completion", () => {
...
-    await triggerInline();
-
-    await emulationUserInteraction();
-
-    verify(
-      stdinMock.write(new SimpleAutocompleteRequestMatcher(), "utf8")
-    ).never();
+    await runSkipIndentInTest("javascript");
+  });
+  it("should skip completion request on Tab key (indention in) in golang where indentation is \t", async () => {
```
Comment (yairco1990) [inline src/test/suite/completion.test.ts:183]: what's the difference between golang and other langs?

File: src/binary/requests/tabSize.ts (hunk 1 above)
Comment (dimacodota) [reply src/binary/requests/tabSize.ts]: my bad 🤦‍♂️

File: src/test/suite/completion.test.ts (hunk 2 above)
Comment (dimacodota) [reply src/test/suite/completion.test.ts:183]: it just will use '\t' by default, it was easier than re-config the IDE indentation in the test

**Review by yairco1990 — APPROVED** [review]

**Pull request merged**